addons/website_sale/models/product_ribbon.py

```diff
@@ -16,8 +20,39 @@ class ProductRibbon(models.Model):
         required=True,
         default='left',
     )
+    style = fields.Selection(
+        string="Style",
+        selection=[('ribbon', "Ribbon"), ('tag', "Badge")],
+        required=True,
+        default='ribbon',
+    )
+    assign = fields.Selection(
```

Comment:
Please add an help on the fields to explain what is `sale`, `new`, and `out-of-stock`